pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -701,6 +722,12 @@ func (oe *operationExecutor) generateMountVolumeFunc(
 				volumeToMount.VolumeSpec.Name(),
 				volumeToMount.PodName,
 				volumeToMount.Pod.UID)
+			oe.recorder.Eventf(volumeToMount.Pod, api.EventTypeNormal, kevents.SuccessfulMountVolume,
```

Comment:
Unmount should only happen once per volume, so a pod should only have one set of unmount events for its volumes, right?

SuccessfulMount I think I would want to see *at least* for non-updating volumes, but I think users will want to see it for all volumes.  I vote to keep it in for now.  It's easy to remove later.